Ginger/GingerCore/ALM/QCCore.cs

```diff
@@ -81,7 +81,13 @@ public override ObservableList<Activity> GingerActivitiesRepo
 
         public override ObservableList<ExternalItemFieldBase> GetALMItemFields(BackgroundWorker bw, bool online, ALM_Common.DataContracts.ResourceType resourceType)
         {
-            return ImportFromQC.GetALMItemFields();
+            ObservableList<ExternalItemFieldBase> tempFieldsList = ImportFromQC.GetALMItemFields();
+            AlmItemFields = new ObservableList<ExternalItemFieldBase>();
```

Comment:
this 5 lines of code are the same for all GetALMItemFields in all ALM types, can we do it in some generic function? can be some function on the base class ALMCore